packages/web-api/package.json

```diff
@@ -77,7 +77,7 @@
     "eslint-plugin-node": "^11.1.0",
     "mocha": "^9.1.0",
     "nock": "^13.1.0",
-    "nyc": "^14.1.1",
+    "nyc": "^15.1.0",
```

Comment:
This is unrelated to the axios upgrade but I've included to resolve the other warnings in `npm audit`.  This is a dev dependency for testing and it should be safe enough.